bin/deploy.sh

```diff
@@ -180,20 +189,21 @@ isalive_check()
     echo "=> OpenAM is alive"
 }
 
-livecheck_stage1()
+import_check()
 {
-    # This livecheck waits for OpenAM config to be imported.
-    # We are looking to amster pod logs periodically.
-    echo "=> Livecheck stage1 - waiting for config to be imported to OpenAM";
+    # This live check waits for OpenAM config to be imported.
```

Comment:
Not a must fix for now, but in general, could you use "AM" instead of "OpenAM" except when referring to the openam pod? Please change globally.

References to our product names in all UIs, docs, marketing materials etc. have moved away from the names starting with "Open", would be best if we moved away from them in comments. (It would also be best if we changed the name of the openam, openig, openidm pods to am, ig, and idm but there's some overhead associated with that that Warren is not willing to incur.)